### ISSUE TITLE
Update Entry

### ISSUE THREAD

**Issue by nicop2000** [Tutorialwork/cloud_kit#7]
How to update an entry?

**Comment by ivofernandes** [comment]
The idea is to delete and save again, but the delete is bugged, will create a pull request next week I guess

**Comment by mattrltrent** [comment]
There really should be an upsert operation.

**Comment by ivofernandes** [comment]
@mattrltrent Is just a delete followed by an add

**Comment by mattrltrent** [comment]
@ivofernandes This isn't atomic though? Let's say you want to *guarantee* your existing key is either updated or left alone. With this current implementation, that seems impossible. This is because the delete could succeed, but saving a new value could fail.

**Comment by ivofernandes** [comment]
I guess for that you need to dive into iOS cloudkit documentation and see if there are other methods besides what this package is using?

**Comment by SethuSenthil** [comment]
Yup, there seems to be a decided update method as documented here: https://developer.apple.com/documentation/cloudkitjs/cloudkit/recordsbatchbuilder/1628739-update
